application/org.openjdk.jmc.flightrecorder.flameview/src/main/js/flameviewColoring.js

```diff
@@ -60,6 +60,9 @@ const packageMarkerRest = "rest";
 const packagesIdentifierMap = new Map().set("java.", packageMarkerJava).set("sun.", packageMarkerSun).set("com.sun.", 
 		packageMarkerComSunAndJdk).set("jdk.", packageMarkerComSunAndJdk);
 const packageColorMap = new Map().set("", rootPackageColor);
+const specialCharactersMap = new Map().set('#','\x23').set('$','\x24').set('(', '\x28').set(')','\x29')
```

Comment:
inconsistencies in spaces after the `,` between the special character and it's alternate value